resources/model/OpenStudio.idd

```diff
@@ -15734,6 +15746,15 @@ OS:Coil:Cooling:DX:TwoSpeed,
        \autosizable
        \units m3/s
        \minimum> 0
+  N5 , \field Unit Internal Static Air Pressure
+       \note Enter pressure drop for the unit containing the coil.
+       \note This value is only used to calculate Energy Efficiency Ratio
+       \note (EER), Integrated Energy Efficiency Ratio (IEER), and the
+       \note Standard Rating (Net) Cooling Capacity. This value is not
+       \note used for modeling the evaporator fan during simulations.
+       \type real
+       \units Pa
+       \minimum> 0.0
```

Comment:
OS:Coil:Cooling:DX:TwoSpeed

This could be changed to be required-field, and ctor assign the apparently-used default listed in the I/O ref https://bigladdersoftware.com/epx/docs/9-3/input-output-reference/group-heating-and-cooling-coils.html#field-unit-internal-static-air-pressure

> If this field is not used, then the standard ratings are still performed but use a default for specific fan power of **773.3 (W/(m3/s))**

@kbenne  please advise

Reply:
I also prefer required field and initialize in ctor in this case.